README.md

```diff
@@ -11,55 +11,84 @@ and several standalone Django apps for specific parts of the site.
 Full installation and usage instructions are available in
 [our documentation](https://cfpb.github.io/consumerfinance.gov).
 
-This project requires Python 3.6, Node 14, and Gulp 4.
-We recommend the use of [virtualenv](https://virtualenv.pypa.io/en/stable/) and
-[virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
-for keeping the project's Python dependencies contained.
+This quickstart requires a working Docker Desktop installation,
```

Comment:
To promote simplicity and avoid duplication, what do you think about removing these instructions from the README in favor of pointing to the docs site? Having the same quickstart content in two places introduces potential for drift, especially because we have to link to the sub-steps in different ways (absolute URLs here, relative in the docs).

Reply:
I think that's reasonable. Duplicating it made sense in an [ARID sort of way](https://www.writethedocs.org/guide/writing/docs-principles/#arid), but I'm fine removing it and linking.